connectors/opa/main.go

```diff
@@ -62,6 +69,27 @@ func RunCmd() *cobra.Command {
 			router.Use(gin.Logger())
 
 			bindAddress := fmt.Sprintf("%s:%d", ip, port)
+			if utils.GetPolicyManagerUseTLS() {
```

Comment:
for TLS we need another commandPort 8443

Reply:
the port when using tls is 443. please see charts/fybrik/notebook-test-readflow.tls.values.yaml

```
katalogConnector:
  service:
    port: 443